qa/purchase_crypto_listing.py

```diff
@@ -0,0 +1,153 @@
+import requests
+import json
+import time
+from collections import OrderedDict
+from test_framework.test_framework import OpenBazaarTestFramework, TestFailure
+
+
+class PurchaseCryptoListingTest(OpenBazaarTestFramework):
+
+    def __init__(self):
+        super().__init__()
+        self.num_nodes = 2
+
+    def run_test(self):
+        alice = self.nodes[0]
+        bob = self.nodes[1]
```

Comment:
Can we adopt more meaningful names in our QA tests which describe the person's role in the transaction? I don't want to have to remember if alice is the buyer or the vendor, let's just call them what they are.